src/getInputs.ts

```diff
@@ -10,6 +10,7 @@ export function getInputs(): Inputs {
   const draft = getInput("draft") ? JSON.parse(getInput("draft")) : undefined;
   const body = getInput("body") || undefined;
   const reviewers = getInput("reviewers");
+  const baseurl = getInput("base_url") || undefined;
```

Comment:
Instead of reading this as an input, I think we can pull the correct value out of the environment: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
addressed this.